lib/node_modules/@stdlib/constants/float64/max-safe-nth-factorial/README.md

```diff
@@ -0,0 +1,164 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# FLOAT64_MAX_SAFE_NTH_FACTORIAL
+
+> Maximum safe nth [Factorial][factorial] when stored in [double-precision floating-point][ieee754] format.
```

Comment:
```suggestion
> Maximum safe nth [factorial][factorial] when stored in [double-precision floating-point][ieee754] format.
```

I think you can lowercase "factorial" here and elsewhere in descriptions in this PR. I don't believe that "factorial" is a proper noun.

Reply:
@kgryte , I have changed them everywhere.